mathml/relations/html5-tree/tabindex-002.html

```diff
@@ -10,6 +10,7 @@
 <script src="/resources/testdriver.js"></script>
 <script src="/resources/testdriver-vendor.js"></script>
 <div id="log"></div>
+<a tabindex="1" href="#link">tabindex(1)-html</a>
```

Comment:
Why the tabindex="1" here?  The mtext that's a link doesn't have that...

Reply:
That was to make the HTML link focused before *all* the other focusable MathML elements in this test. This explicit tabindex="1" does not make the link tab-navigable on macOS. 

However, it's true that we actually only need that HTML link to be focused before the mtext link and removing the tabindex attribute make them more similar. So I'll to remove it.